src/main/java/mate/academy/AsyncRequestProcessor.java

```diff
@@ -1,16 +1,32 @@
 package mate.academy;
 
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
 
 public class AsyncRequestProcessor {
+    private static final int SLEEP_TIME = 200;
```

Comment:
The constant `SLEEP_TIME` should have a more descriptive name indicating its purpose. For example, `SIMULATED_DELAY_MS` might be clearer in indicating that this is a simulated delay for the demo.

Reply:
agree, I'd also add info at least about the time units that it is in, like:
```suggestion
    private static final int SLEEP_TIME_MILLIS = 200; // or MS
```